### PR TITLE
refactor, chore(egen): Replaces K80 GPU with T4, kfp and tensorflow versoin updates, pre-built Docker container image for training and prediction update, other corrections from template

### PR DESCRIPTION
**Pull request by SumanthKasula99**
This PR consists of the following updates:

1. Replaces K80 GPU with Tesla T4 GPU.
2. Updates kfp to 2.7.0 and tensorflow to 2.15.1
3. Updates pre-built Docker container image for training and prediction to 2.13
4. Refactors the notebook according to the notebook template (removes boilerplate elements, renames REGION with LOCATION and re-structures the sections).
5. Adds link for Colab enterprise.
6. Removes the usage of future tense(will) according to the guidelines.
7. Cleans up the intermediates files created.

**REQUIRED:** Fill out the below checklists or remove if irrelevant
1. If you are opening a PR for `Official Notebooks` under the [notebooks/official](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/notebooks/official) folder, follow this mandatory checklist:
- [x] Use the [notebook template](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/notebook_template.ipynb) as a starting point.
- [x] Follow the style and grammar rules outlined in the above notebook template.
- [x] Verify the notebook runs successfully in Colab since the automated tests cannot guarantee this even when it passes.
- [x] Passes all the required automated checks. You can locally test for formatting and linting with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).
- [ ] You have consulted with a tech writer to see if tech writer review is necessary. If so, the notebook has been reviewed by a tech writer, and they have approved it.
- [ ] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/official/CODEOWNERS) file under the `Official Notebooks` section, pointing to the author or the author's team.
- [x] The Jupyter notebook cleans up any artifacts it has created (datasets, ML models, endpoints, etc) so as not to eat up unnecessary resources.

### CODE REVIEW

File: notebooks/official/pipelines/get_started_with_machine_management.ipynb [hunk 1]
```diff
@@ -555,19 +505,19 @@
         "\n",
         "Next, set the machine type to use for training and prediction.\n",
         "\n",
-        "- Set the variables `TRAIN_COMPUTE` and `DEPLOY_COMPUTE` to configure  the compute resources for the VMs you will use for for training and prediction.\n",
+        "- Set the variables `TRAIN_COMPUTE` and `DEPLOY_COMPUTE` to configure  the compute resources for the VMs you use for for training and prediction.\n",
```
Comment (katiemn) [inline notebooks/official/pipelines/get_started_with_machine_management.ipynb:508]: remove one 'for'

File: notebooks/official/pipelines/get_started_with_machine_management.ipynb [hunk 2]
```diff
@@ -72,20 +75,20 @@
       "source": [
         "### Objective\n",
         "\n",
-        "In this tutorial, you learn how to convert a self-contained custom training component into a `Vertex AI CustomJob`, whereby:\n",
+        "In this tutorial, you learn how to convert a self-contained custom training component into a Vertex AI CustomJob, whereby:\n",
```
Comment (kittyabs) [inline notebooks/official/pipelines/get_started_with_machine_management.ipynb:78]: Add back ticks to CustomJob:

- Vertex AI `CustomJob`

File: notebooks/official/pipelines/get_started_with_machine_management.ipynb [hunk 3]
```diff
@@ -72,20 +75,20 @@
...
         "\n",
-        "- `Vertex AI Pipelines`\n",
+        "- Vertex AI Pipelines\n",
         "\n",
         "The steps performed in this tutorial include:\n",
         "\n",
         "- Create a custom component with a self-contained training job.\n",
         "- Execute pipeline using component-level settings for machine resources\n",
-        "- Convert the self-contained training component into a `Vertex AI CustomJob`.\n",
+        "- Convert the self-contained training component into a Vertex AI CustomJob.\n",
```
Comment (kittyabs) [inline notebooks/official/pipelines/get_started_with_machine_management.ipynb:91]: Add back ticks to CustomJob:

- Vertex AI `CustomJob`

File: notebooks/official/pipelines/get_started_with_machine_management.ipynb [hunk 4]
```diff
@@ -479,16 +429,16 @@
         "\n",
         "You can set hardware accelerators for training and prediction.\n",
         "\n",
-        "Set the variables `TRAIN_GPU/TRAIN_NGPU` and `DEPLOY_GPU/DEPLOY_NGPU` to use a container image supporting a GPU and the number of GPUs allocated to the virtual machine (VM) instance. For example, to use a GPU container image with 4 Nvidia Telsa K80 GPUs allocated to each VM, you would specify:\n",
+        "Set the variables `TRAIN_GPU/TRAIN_NGPU` and `DEPLOY_GPU/DEPLOY_NGPU` to use a container image supporting a GPU and the number of GPUs allocated to the virtual machine (VM) instance. For example, to use a GPU container image with 4 Nvidia Telsa T4 GPUs allocated to each VM, you would specify:\n",
```
Comment (kittyabs) [inline notebooks/official/pipelines/get_started_with_machine_management.ipynb:432]: remove "would":

- From: "you would specify"
- To: "you specify"

File: notebooks/official/pipelines/get_started_with_machine_management.ipynb [hunk 5]
```diff
@@ -479,16 +429,16 @@
...
-        "    (aiplatform.gapic.AcceleratorType.NVIDIA_TESLA_K80, 4)\n",
+        "    (aiplatform.gapic.AcceleratorType.NVIDIA_TESLA_T4, 4)\n",
         "\n",
         "\n",
         "Otherwise specify `(None, None)` to use a container image to run on a CPU.\n",
         "\n",
         "Learn more about [hardware accelerator support for your region](https://cloud.google.com/vertex-ai/docs/general/locations#accelerators).\n",
         "\n",
-        "*Note*: TF releases before 2.3 for GPU support will fail to load the custom model in this tutorial. It is a known issue and fixed in TF 2.3. This is caused by static graph ops that are generated in the serving function. If you encounter this issue on your own custom models, use a container image for TF 2.3 with GPU support."
+        "*Note*: TF releases before 2.3 for GPU support fails to load the custom model in this tutorial. It is a known issue and fixed in TF 2.3. This is caused by static graph ops that are generated in the serving function. If you encounter this issue on your own custom models, use a container image for TF 2.3 with GPU support."
```
Comment (kittyabs) [inline notebooks/official/pipelines/get_started_with_machine_management.ipynb:441]: contract "It is" -> "It's"

**Review by kittyabs — COMMENTED** [review]
numerous edits

File: notebooks/official/pipelines/get_started_with_machine_management.ipynb (hunk 1 above)
Comment (katiemn) [reply notebooks/official/pipelines/get_started_with_machine_management.ipynb:508]: Instead, it should be: - Set the variables `TRAIN_COMPUTE` and `DEPLOY_COMPUTE` to configure  the compute resources for the VMs you use for training and prediction.

**Review by katiemn — APPROVED** [review]

**Pull request merged**